### DISCUSSION TITLE
Optional SPAM protection

### DISCUSSION THREAD

**Discussion by gbdesign2023** [tobimori/kirby-dreamform#71] [Feature Requests]
I have built an additional SPAM protection into my forms, which could perhaps be a great feature for your tool: 
A simple maths question whose solution must be selected - with two additional, arbitrary values - in a dropdown. You hardly have to do any arithmetic, as the result is obviously recognisable because the other results are too far apart.

- The correct number is linked to the send button. The button is only displayed once the correct result has been selected.
- The position of the numbers in the dropdown is constantly changed with Shuffle
- Javascript is required and thus keeps additional bots away. 

https://github.com/tobimori/kirby-dreamform/assets/140849227/1e43c97b-4e11-463c-82a4-ce14fe77cc95

**Comment by tobimori** [comment]
I'd recommend you to publish your solution as plugin integrating into DreamForm, let me know if you need help on that! 

I'm currently hestitant on including solutions in Core that:
- need custom JS code
- don't work with cached sites
